base/base-guide.md

```diff
@@ -0,0 +1,151 @@
+# {Doctype} guide
+
+This {doctype}-guide provides extra writing tips for each of the sections within the {doctype}-template.
+
+**Version:** {MAJOR.MINOR.PATCH}
+
+**Last updated:** {Month DD, YYYY}
+
+## Prerequisites
+
+To make the best use of this guide, it helps if you have a working knowledge of:
+
+* The project's documentation style guide.
```

Comment:
**Question:** The Good Docs Project's style guide or the destination project's style guide?

Reply:
Good question. I'll change to "The {Project/Google/Microsoft/...} documentation style guide."